internal/descriptor/registry.go

```diff
@@ -582,3 +605,111 @@ func (r *Registry) packageIdentityName(f *descriptorpb.FileDescriptorProto) stri
 	}
 	return f.GetPackage()
 }
+
+// RegisterOpenAPIOptions registers OpenAPI options
+func (r *Registry) RegisterOpenAPIOptions(opts *apiconfig.OpenAPIOptions) error {
+	if opts == nil {
+		return nil
+	}
+
+	for _, opt := range opts.File {
+		if _, ok := r.files[opt.File]; !ok {
+			return fmt.Errorf("no file %s found", opt.File)
```

Comment:
I think these errors should instead be warnings, like when you add an HTTP for a non-existent method. Is that correct?

Reply:
I misremebered and unbound HTTP rules are errors (https://github.com/grpc-ecosystem/grpc-gateway/blob/v2/protoc-gen-grpc-gateway/main.go#L112).